database/couchdb.js

```diff
@@ -20,7 +20,7 @@ const seed = async (database) => {
           const caption = faker.commerce.productName();
           const is_primary = true;
           roomPhotos.push({ storage_url, name, caption, is_primary });
-          const room_number = (run + 1) * i;
+          const room_number = (run + i + j);
           docs.push({room_number, photos: roomPhotos});
```

Comment:
This looks good. Each room will have 5 photos associated with it?

Reply:
yes, 5 photos with each room and one marked as primary